data/mods/DinoMod/items/pets_elephant_armor.json

```
@@ -0,0 +1,142 @@
+        "text": "A full barding for elephants consisting of boiled leather and cloth undercovering.  This caparison is depicting a battle between a monstrous demon and a regal warrior astride an elephant and wielding an elaborate vajra.  You could put this on a friendly elephant."
+      },
+      {
+        "id": "khajurahoshiva",
+        "text": "A full barding for elephants consisting of boiled leather and cloth undercovering.  This caparison is depicting a god with many arms bearing different items with one large uncovered breast on the left side.  You could put this on a friendly elephant."
+      },
+      {
+        "id": "elephantry",
+        "text": "A full barding for elephants consisting of boiled leather and cloth undercovering.  This caparison is depicting several charging elephantry trampling enemy soldiers.  You could put this on a friendly elephant."
+      },
+      {
+        "id": "diamond",
+        "text": "A full barding for elephants consisting of boiled leather and cloth undercovering.  This caparison is depicting an assortment of diamonds and gems.  You could put this on a friendly elephant."
+      },
+      {
+        "id": "ashoka",
+        "text": "A full barding for elephants consisting of boiled leather and cloth undercovering.  This caparison is depicting a man in rich garments with a full uncovered belly on an elaborate chariot.  You could put this on a friendly elephant."
+      },
+      {
+        "id": "singh",
+        "text": "A full barding for elephants consisting of boiled leather and cloth undercovering.  This caparison is depicting several identical smiling men in turbans of various bright colors performing a dance.  You could put this on a friendly elephant."
+      },
+      {
+        "id": "triceratops",
+        "text": "A full barding for elephants consisting of boiled leather and cloth undercovering.  This caparison is depicting a fierce three horned dinosaur bursting out the side of a 1970s van.  You could put this on a friendly elephant."
+      }
+    ]
+  },
+  {
+    "type": "PET_ARMOR",
+    "id": "leatherbone_armor_elephant",
+    "copy-from": "elephant_armor",
+    "color": "green",
+    "name": { "str": "boiled leather elephant barding with bones", "str_pl": "boiled leather elephant bardings with bones" },
+    "description": "Decorative bones affixed to leather elephant barding to invoke fear in bandits and raiders and traders all!  You could put this on a friendly elephant.",
+    "price": 45000,
+    "price_postapoc": 3000,
+    "material": [ "bone", "leather" ],
+    "weight": "17 kg",
+    "volume": "150 L",
+    "material_thickness": 6
+  },
+  {
+    "type": "PET_ARMOR",
+    "id": "rubber_armor_elephant",
+    "copy-from": "elephant_armor",
+    "color": "green",
+    "name": { "str": "elephant rain sheet" },
+    "description": "A thin plastic covering adapted for elephant to protect from acid rain and other caustic sources.  You could put this on a friendly elephant.",
```

```suggestion
    "description": "A thin plastic covering, adapted to protect an elephant from acid zombie splashes and similar caustic substances.  You could put this on a friendly elephant.",
```
CDDA no longer has acid rain?